jetty-core/jetty-alpn/jetty-alpn-conscrypt-server/src/test/java/org/eclipse/jetty/alpn/conscrypt/server/ConscryptHTTP2ServerTest.java

```diff
@@ -140,4 +139,12 @@ public void testSimpleRequest() throws Exception
             assertEquals(200, contentResponse.getStatus());
         }
     }
+
+    @Test
+    public void testSNIRequired() throws Exception
+    {
+        // The KeyStore contains 1 certificate with two DNS names.
+        httpsConfig.getCustomizer(SecureRequestCustomizer.class).setSniRequired(true);
```

Comment:
This modifies the `httpsConfig` instance for all tests of this class ran after this one.

A fresh `httpsConfig` should be created for each test.

Reply:
No, I just checked, and JUnit allocates a new instance for every test method. This is evident also from the logs that print different hash codes for each `Server` instance:
```
...
2024-11-19 18:57:46.602:INFO :oejs.Server:main: Started oejs.Server@13bc8645{STARTING}[12.0.16-SNAPSHOT,sto=0] @824ms
...
2024-11-19 18:57:46.815:INFO :oejs.Server:main: Started oejs.Server@34f22f9d{STARTING}[12.0.16-SNAPSHOT,sto=0] @1037ms
```